tomb-common/src/banyan_api/models/snapshot.rs

```diff
@@ -19,6 +20,18 @@ pub struct Snapshot {
     pub created_at: i64,
 }
 
+impl Display for Snapshot {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
Hah I do this all the time (including the full path to the module) super useful for quickly moving code around. 👍🏼